src/controller/python/chip/clusters/Objects.py

```diff
@@ -14749,11 +14749,12 @@ class LockDataTypeEnum(MatterIntEnum):
             kRfid = 0x07
             kFingerprint = 0x08
             kFingerVein = 0x09
+            kFace = 0x0A
             # All received enum values that are not listed above will be mapped
             # to kUnknownEnumValue. This is a helper enum value that should only
             # be used by code to process how it handles receiving and unknown
             # enum value. This specific should never be transmitted.
-            kUnknownEnumValue = 10,
+            kUnknownEnumValue = 11,
```

Comment:
why not 0x0B?

Reply:
Because this is generated code and the generator maintainer has decided to use decimal values.